lib/OpenQA/WebAPI.pm

```diff
@@ -53,6 +53,7 @@ sub startup ($self) {
 
     # register basic routes
     my $logged_in = $r->under('/')->to('session#ensure_user');
+    my $auth_any_user = $r->under('/')->to('Auth#auth');
```

Comment:
I notice that we have now two equal routes. one here `$aut_any_user` and one in L68 `$api_auth_any_user`. I dont know for sure but I have a feeling that they can be ust one. wdys?

Reply:
The other route is under `$r->any('/api/v1')`. So no, they are no equivalent. It would maybe be possible to reduce redundancy. However, I don't think it is worthwhile to change how the routing is structured for the sake of this.